components/server/tests/external/routes/test_source.py

```diff
@@ -174,7 +174,7 @@ def assert_url_check(self, response, status_code: int = None, status_code_reason
         availability = dict(
             status_code=status_code, reason=status_code_reason, source_uuid=SOURCE_ID, parameter_key="url"
         )
-        self.assertEqual(dict(ok=True, availability=[availability]), response)
+        self.assertEqual(dict(ok=True, availability=[availability], affected=1), response)
```

Comment:
```suggestion
        self.assertEqual(dict(ok=True, availability=[availability], nr_sources_updated=1), response)
```

And so on for the other tests below.